Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -356,55 +371,35 @@ extension AttributedStringSerializer {
 
         return nil
     }
-}
 
+}
 
-// MARK: - Implicit Representations
-//
 private extension AttributedStringSerializer {
 
-    /// Some elements have an implicit representation that doesn't really follow the standard
-    /// conversion logic.  This method takes care of such specialized conversions.
-    ///
-    /// - Parameters:
-    ///     - element: the element to request an implicit representation for.
-    ///     - attributes: the attributes that the output attributed string will inherit.
-    ///
-    /// - Returns: the requested implicit representation, if one exists, or `nil`.
-    ///
-    func implicitRepresentation(for element: ElementNode, inheriting attributes: [AttributedStringKey: Any]) -> NSAttributedString? {
+    // MARK: - Element Converters
 
-        guard let elementType = element.standardName else {
+    /// Some element types have an implicit representation that doesn't really follow the standard
+    /// conversion logic.  Element Converters take care of that.
+    func converter(for element: ElementNode) -> ElementConverter? {
+        guard let standardType = element.standardName else {
             return nil
         }
 
-        return implicitRepresentation(for: elementType, inheriting: attributes)
-    }
-
-
-    /// Some element types have an implicit representation that doesn't really follow the standard
-    /// conversion logic.  This method takes care of such specialized conversions.
-    ///
-    /// - Parameters:
-    ///     - elementType: the element type to request an implicit representation for.
-    ///     - attributes: the attributes that the output attributed string will inherit.
-    ///
-    /// - Returns: the requested implicit representation, if one exists, or `nil`.
-    ///
-    private func implicitRepresentation(for elementType: StandardElementType, inheriting attributes: [AttributedStringKey: Any]) -> NSAttributedString? {
+        let equivalentNames = standardType.equivalentNames
 
-        switch elementType {
-        case .hr, .img, .video:
-            return NSAttributedString(string: .textAttachment, attributes: attributes)
-        case .br:
-            return NSAttributedString(.lineSeparator, attributes: attributes)
-        default:
-            return nil
+        for (key, converter) in elementConvertersMap {
+            if equivalentNames.contains(key.rawValue) {
```

Comment:
I really really like what you did here. Even considering the Naming equivalences!